tensorflow/contrib/tensorrt/python/trt_convert.py

```diff
@@ -137,6 +144,7 @@ def create_inference_graph(input_graph_def,
                            is_dynamic_op=False,
                            maximum_cached_engines=1,
                            cached_engine_batch_sizes=None,
+                           use_calibration=True,
```

Comment:
Please add a test, maybe using the mnist example.

Reply:
+1

Reply:
I've added some tests. I also have an example script which can be used to train and test a simple model on MNIST - should I include that as well? Where would it go?